entc/gen/graph.go

```diff
@@ -643,17 +644,25 @@ func (a assets) write() error {
 
 // format runs "goimports" on all assets.
 func (a assets) format() error {
-	for _, file := range a.files {
-		path := file.path
-		src, err := imports.Process(path, file.content, nil)
-		if err != nil {
-			return fmt.Errorf("format file %s: %v", path, err)
-		}
-		if err := ioutil.WriteFile(path, src, 0644); err != nil {
-			return fmt.Errorf("write file %s: %v", path, err)
-		}
+	var eg errgroup.Group
+
+	for i := range a.files {
+		idx := i
+		f := a.files[idx]
+		eg.Go(func() error {
+			path := f.path
+			src, err := imports.Process(path, f.content, nil)
+			if err != nil {
+				return fmt.Errorf("format file %s: %v", path, err)
```

Comment:
I think a better format here might be something like "issue with processing path '%v': %w", path, err

Reply:
It's actually formatting a file, not processing a path. Perhaps a better alternative is `formatting file %s: %w` which also includes error unwrapping which is supposed to be there anyway. Generally error messages, because they're wrapped, don't include "error doing xyz" as that's extraneous if they all  have that.

Another PR which I'll open will improve the error wrapping in entc, as currently they're just logged with fatal.

https://github.com/facebook/ent/blob/8b9479599d754d6e66ab5bc19885b4091c660b3f/cmd/internal/base/base.go#L162